<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogViewer</class>
 <widget class="QWidget" name="LogViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="typeLabel">
     <property name="styleSheet">
      <string notr="true">font-size: 17px;</string>
     </property>
     <property name="text">
      <string>Assignment Type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font-size: 13px;</string>
     </property>
     <property name="text">
      <string>Standard Output</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="outputView">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">font-size: 13px;</string>
     </property>
     <property name="text">
      <string>Standard Error</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="errorView">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
